tasks/build_tags.py

```diff
@@ -2,6 +2,7 @@
 Utilities to manage build tags
 """
 import sys
+from typing import List
```

Comment:
We have some builders still using Python 3.6 which run invoke tasks. I wasn't sure whether that would break or not (and I'd need a deploy pipeline to check that, since most of the jobs that use old Python jobs are deploy jobs), so I went with the safe option here.

I'll add a todo to check if that's really needed.

Reply:
👍 I think Python 3.6 doesn't support this, no